frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/advanced-section.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Checkbox,
+  CheckboxProps,
+  ExpandableSection,
+  ExpandableSectionProps,
+  FormGroup,
+} from '@patternfly/react-core';
+import { DeploymentType } from '../../../../constants/create-storage-system';
+import { WizardDispatch, WizardState } from '../../reducer';
+import './backing-storage.scss';
+
+const RADIO_GROUP_NAME = 'deployment-type';
+
+export const AdvancedSection: React.FC<AdvancedSelectionProps> = ({
```

Comment:
I am still considering whether we need this section just for this one option. Seems like overdoing it. Will we be adding more options in future PRs ?

Reply:
Yes we want to add more. No doubt in that.
Its advanced because general users dont want that